ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/PullQueryPublisherTest.java

```diff
@@ -80,10 +81,11 @@ public void setUp() {
     publisher = new PullQueryPublisher(
         serviceContext,
         statement,
-        pullQueryExecutor);
+        pullQueryExecutor,
+        0L);
```

Comment:
Can we go with a non-zero start time please?  Stick it in a constant, e.g. `START_TIME = 1234L`. Update tests to use `START_TIME`.